src/json/check_config.cpp

```diff
@@ -0,0 +1,82 @@
+#include "utils_json.hpp"
+#include <climits>
+#include <iostream>
+
+#define DEFAULT_SERVER_NAME "default_server_name"
+
+namespace json
+{
+
+static int
+check_value_servername(t_object *config)
+{
+	if (config->find("server_name") == config->end())
+	{
+		(*config)["server_name"] = json::Value(new std::string(DEFAULT_SERVER_NAME));
+		return 0;
+	}
+	else
+	{
+		if (config->at("server_name").get_type() != JSON_STRING)
+		{
+			std::cerr << "Error: the server_name is not a string" << std::endl;
+			return 1;
+		}
+	}
```

Comment:
why not do an else if instead of an else followed by an if ?
```suggestion
	else if (config->at("server_name").get_type() != JSON_STRING)
	{
		std::cerr << "Error: the server_name is not a string" << std::endl;
		return 1;
	}
```

Reply:
Nice comment,
done: [26b845c](https://github.com/t-h2o/webserv/pull/49/commits/26b845c3dc2a9719d742c9c4a37223e7810cab3f)